config/routes.rb

```diff
@@ -146,6 +149,10 @@
         end
       end
 
+      resources :lectures, concerns: :paginatable do
+        get :access_link, on: :member
```

Comment:
Don't need the `on: :member` if I remember correctly. You can remove it and verify by running `rake routes`.